packages/postcss-unique-selectors/src/index.js

```diff
@@ -1,11 +1,25 @@
 import { plugin } from 'postcss';
 import sort from 'alphanum-sort';
 import uniqs from 'uniqs';
+import selectorParser from 'postcss-selector-parser';
+
+function parseSelectors(selectors, callback) {
+  return selectorParser(callback).processSync(selectors);
+}
 
 function unique(rule) {
   rule.selector = sort(uniqs(rule.selectors), { insensitive: true }).join();
 }
 
 export default plugin('postcss-unique-selectors', () => {
-  return (css) => css.walkRules(unique);
+  return (css) =>
+    css.walkRules((nodes) => {
+      let comments = [];
+      nodes.selector = parseSelectors(nodes.selector, (selNode) => {
+        selNode.walkComments((sel) => comments.push(sel.value));
```

Comment:
I think it's better to do one tree walk instead of two here:

```js
selNode.walkComments((sel) => {
  comments.push(sel.value);
  sel.remove();
})
```

Reply:
oohhh....silly me 😄 
it could have done in single walk...
I Will change that soon 👍

Reply:
done 👍